src/runtime.js

```diff
@@ -179,10 +179,11 @@ function adoptShared(global, opts, callback) {
   if (!global.AMP.extension) {
     /**
      * @param {string} unusedName
+     * @param {string} version
      * @param {function(!Object)} installer
      * @const
      */
-    global.AMP.extension = function(unusedName, installer) {
+    global.AMP.extension = function(unusedName, version, installer) {
```

Comment:
It would be good to combine `unusedName` and `version` to a single object, and make the version  to be `0.1` as default. But this can be done afterwards.